admin/resources/js/components/ErrorToast.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useCallback, useContext } from "react";
+import { ErrorContext } from "./ErrorContainer";
+import Alert from "./H2Components/Alert";
+
+export const ErrorToast: React.FC = () => {
```

Comment:
I think all this ErrorToast functionality should be part of ErrorContainer. I can't think of a reason we'd use the former without the latter.